tests_react_native/src/index.tsx

```diff
@@ -0,0 +1,817 @@
+/*
```

Comment:
I don't understand why there are 817 new lines of typescript that appears to duplicate the current example index page? That duplication is not going to be maintainable I don't think

Reply:
Just temporarily - forcing stuff to run 😅